api/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
Is this redundant with line #2?

Reply:
not sure, looks like auto-generated reverted.